src/node_pch.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#define NAPI_EXPERIMENTAL
```

Comment:
Is this always safe? I see in some files where this is defined other headers are included before. With this here, `NAPI_EXPERIMENTAL` will always be defined, can probably be removed from everywhere in the code.

There is also `NODE_WANT_INTERNALS`. There is and explicit define in `inspector_socket.cc` which will be ignored by all headers pulled by these here.

This might be safe, but is not obvious to me. Might also become difficult to maintain, since anyone not using Windows will only see errors when running CI (if at all).

Reply:
Removed.